src/Order.Data/Repositories/IOrderStatusRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Order.Data.Entities;
+using Order.Data.Specifications;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Order.Data
+{
+    public interface IOrderStatusRepository
```

Comment:
I prefer to do use a separate repository for each entity or Aggregate in DDD. From Aggregate perspective might be this repository redundant and the method can be part of OrderRepository.